pkgs/development/tcl-modules/by-name/ji/jimlib/package.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  fetchFromGitLab,
+  mkTclDerivation,
+}:
+
+mkTclDerivation rec {
+  pname = "jimlib";
+  version = "0.15.0-unstable-2024-10-31";
+
+  src = fetchFromGitLab {
+    owner = "dbohdan";
+    repo = "jimlib";
+    rev = "6a50339a6fbfc26d793b987c939cdd82dc1db490";
+    sha256 = "sha256-EkYjaQqH9I8SCS424cpsyfigPweLLLLPvRkO+vHHCJ8=";
+  };
+
+  dontBuild = true;
+  doCheck = true;
+
+  checkPhase = ''
+    tclsh ./jimlib.tcl
+  '';
+
+  installPhase = ''
+    mkdir -p $out/lib/jimlib
+    cp $src/jimlib.tcl $out/lib/jimlib/jimlib.tcl
+    cp $src/pkgIndex.tcl $out/lib/jimlib/pkgIndex.tcl
+  '';
+
+  meta = with lib; {
+    homepage = "https://wiki.tcl-lang.org/page/jimlib";
```

Comment:
I'm not sure the wiki page can be considered the official homepage. I'd use the gitlab page instead. Up to you.

Reply:
Gitlab makes sense to me, updated